From a4cbbd907fa765a9115cc986accf414a53e39e7f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 16 Sep 2019 08:57:18 -0700
Subject: [PATCH] TOOLS-2254 add redirects from mo-docs to modocs.joyent.us
 (restdown files should now always be .md) Reviewed by: Tim Foster
 <tim.foster@joyent.com> Approved by: Tim Foster <tim.foster@joyent.com>

---
 prototypes/prototype.restdown | 15 ---------------
 1 file changed, 15 deletions(-)
 delete mode 100644 prototypes/prototype.restdown

diff --git a/prototypes/prototype.restdown b/prototypes/prototype.restdown
deleted file mode 100644
index 3bec307..0000000
--- a/prototypes/prototype.restdown
+++ /dev/null
@@ -1,15 +0,0 @@
----
-title: Something nice
-markdown2extras: wiki-tables, code-friendly
-apisections: Something, Useful
----
-<!--
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
--->
-
-<!--
-    Copyright 2019 Joyent, Inc.
--->
-
-- 
2.21.0

